packages/docusaurus-plugin-content-blog/package.json

```
@@ -28,6 +28,7 @@
     "feed": "^4.2.2",
     "fs-extra": "^10.0.0",
     "globby": "^11.0.2",
+    "js-yaml": "^4.1.0",
```

```suggestion
    "js-yaml": "^4.0.0",
```

use same semver range as other package